article.md

```diff
@@ -173,15 +232,17 @@ async def listing_handler(context: PlaywrightCrawlingContext) -> None:
 
     await context.page.wait_for_load_state('load')
 
-    job_title = await context.page.locator('//*[@id="main-content"]/section[1]/div/section[2]/div/div[1]/div/h1').text_content()
-
-    company_name  = await context.page.locator('//*[@id="main-content"]/section[1]/div/section[2]/div/div[1]/div/h4/div[1]/span[1]/a').text_content()
+    job_title = await context.page.locator('div.top-card-layout__entity-info h1.top-card-layout__title').text_content()
+   
+    company_name  = await context.page.locator('span.topcard__flavor a').text_content()   
 
-    time_of_posting= await context.page.locator('//*[@id="main-content"]/section[1]/div/section[2]/div/div[1]/div/h4/div[2]/span').text_content()
+    time_of_posting= await context.page.locator('div.topcard__flavor-row span.posted-time-ago__text').text_content()
 
 
     await context.push_data(
         {
+            # we are maing use of regex to remove special characters for the extracted texts
```

Comment:
Even if I ignore the fact that the sentence is terribly misspelled...

The question was, why do we need to remove special characters from the texts?  The fact that the regex removes special characters is clear. The point of comments is to explain what the reader cannot see from the code.